doc/manuals/admin/perf_tuning.md

```diff
@@ -61,7 +70,7 @@ Orion startup or when entities are created.
 * An index with `expireAfterSeconds: 0` for the `expDate` field in the entities collection, due to functional needs of the [transient entities functionality](../user/transient_entities.md).
 
 You can find an analysis about the effect of indexes in [this document](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/extra/indexes_analysis.md), although
-it is based on an old Orion version, so it is probably outdated.
+it is based on an old Orion version, so it is probably outdated. In addition, please find some useful references about [compound indexes](https://docs.mongodb.com/manual/core/index-compound/) and [sort results with indexes](https://docs.mongodb.com/manual/tutorial/sort-results-with-indexes/) from MongoDB official documentation.
```

Comment:
from **the** MongoDB official documentation   OR:
from **the** official documentation of MongoDB

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd